tpl/fmt/fmt_test.go

```diff
@@ -0,0 +1,14 @@
+package fmt
+
+import (
+	jww "github.com/spf13/jwalterweatherman"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestErrorf(t *testing.T) {
+	errorCount := jww.LogCountForLevel(jww.LevelError)
+	ns := &Namespace{}
```

Comment:
Use the `New` method to create a namespace and make sure the logger (see comment above) is created in that one.